.jenkins/custom_label.Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+@Library("OpenEnclaveCommon") _
+oe = new jenkins.common.Openenclave()
+
+// The below timeout is set in minutes
+GLOBAL_TIMEOUT = 240
+// ctest timeout is set in seconds
+CTEST_TIMEOUT = 480
```

Comment:
We should do this as a separate issue (because it applies to several different files I imagine), but maybe we should rename the TIMEOUT variables to have their time types in them. So for example:

GLOBAL_TIMEOUT becomes GLOBAL_TIMEOUT_MINUTES
and CTEST_TIMEOUT becomes CTEST_TIMEOUT_SECONDS

Then we can get rid of those comments :)

Reply:
This is a very good suggestion.

Reply:
I added a new commit to this PR with your proposed changes